consoles/VNC.pm

```diff
@@ -131,15 +131,20 @@ sub login {
     my $endtime = time + $connect_timeout;
 
     my $socket;
+    my $err_cnt = 0;
     while (!$socket) {
         $socket = IO::Socket::INET->new(
             PeerAddr => $hostname,
             PeerPort => $port,
             Proto    => 'tcp',
         );
         if (!$socket) {
+            $err_cnt++;
             return if (time > $endtime);
-            bmwqemu::diag "Error connecting to host <$hostname>: $@";
+            # we might be too fast trying to connect to the VNC host (e.g.
+            # qemu) so ignore the first occurences of a failed
+            # connection attempt.
+            bmwqemu::diag "Error connecting to host <$hostname>: $@" if ($err_cnt > 2);
```

Comment:
for next time: you don't need the extra () if the condition is simple and post (it shouldn't be post if it's not simple - anyway).